editor/src/clj/editor/debug_view.clj

```diff
@@ -611,31 +612,59 @@
             (current-session debug-view evaluation-context))
   (run [debug-view] (mobdebug/exit! (current-session debug-view))))
 
-(defn- can-change-resolution? [debug-view prefs evaluation-context]
-  (and (targets/controllable-target? (targets/selected-target prefs))
-       (not (suspended? debug-view evaluation-context))))
+(defn- should-rotate-device? [prefs workspace]
+  (prefs/get-prefs prefs (prefs/make-project-specific-key "should-rotate-device" workspace) false))
```

Comment:
Is the `should-rotate-device` preference fully related to the "Simulate Resolution" feature? In that case we might want to name it something like `simulate-rotated-device` to make that explicit. 🤔